torch/onnx/_internal/fx/passes/shape_inference.py

```diff
@@ -19,14 +19,10 @@ class ShapeInferenceWithFakeTensor(_pass.Transform):
     def _run(self, *args, **kwargs) -> torch.fx.GraphModule:
         assert not kwargs, "`kwargs` is not supported."
 
-        # NOTE: torch.fx.Transformer makes a copy of the graph only but shares weights
-        # with the original module.
-        module = torch.fx.Transformer(self.module).transform()
-
         # NOTE(titaiwang): Usually fx graph should have all the node meta value we need,
         # so we don't have to run FakeTensorProp to fill in node meta values. However, this
-        # can be used to validate op-level debugging when we only have symbolic shapes in
-        # graph
+        # is used to fill in static shapes, which are needed by op-level debugging,
+        # when we only have symbolic shapes in graph
 
         # Use this FakeTensorMode to
         # 1. convert nn.Parameter's in nn.Module to FakeTensor
```

Comment:
FYI, https://github.com/pytorch/pytorch/pull/97186 has an update here. It shouldn't matter for this PR, but just in case.

Reply:
Thanks!